docker/pdo_client.dockerfile

```diff
@@ -66,10 +69,13 @@ COPY --chown=${UNAME}:${UNAME} tools/*.sh ./
 RUN /project/pdo/tools/build_client.sh
 
 ARG PDO_HOSTNAME
-ENV PDO_HOSTNAME=$PDO_HOSTNAME
+ENV PDO_HOSTNAME=${PDO_HOSTNAME}
 
 ARG PDO_LEDGER_URL
-ENV PDO_LEDGER_URL=$PDO_LEDGER_URL
+ENV PDO_LEDGER_URL=${PDO_LEDGER_URL}
+
+ARG PDO_LEDGER_KEY_ROOT=/project/pdo/xfer/ccf/keys/
+ENV PDO_LEDGER_KEY_ROOT=${PDO_LEDGER_KEY_ROOT}
 
```

Comment:
same. we control the layout of the container file system. we choose the structure.

Reply:
see above